playgrounds/react-vite/src/tabs/Collectable.tsx

```diff
@@ -222,27 +290,47 @@ function OrdersTable({
   }
 
   return (
-    <Table>
-      <TableHeader>
-        <TableRow>
-          <TableHead>Price</TableHead>
-          <TableHead>Currency</TableHead>
-          <TableHead>{type === "listings" ? "Seller" : "Buyer"}</TableHead>
-          <TableHead>Expiration</TableHead>
-          <TableHead>Actions</TableHead>
-        </TableRow>
-      </TableHeader>
-      <TableBody>
-        {items.map((item) => (
-          <OrdersTableRow
-            key={item.orderId}
-            order={item}
-            actionLabel={actionLabel}
-            onAction={onAction}
-          />
-        ))}
-      </TableBody>
-    </Table>
+    <>
+      <Table>
+        <TableHeader>
+          <TableRow>
+            <TableHead>Price</TableHead>
```

Comment:
Apparently, we don't always get formatted price using as:
`//OrdersTableRow`
`<TableCell>{order.priceAmountFormatted}</TableCell>`

what about using `formatUnits` from viem instead?

<img width="717" alt="Ekran Resmi 2024-11-28 12 02 19" src="https://github.com/user-attachments/assets/86a3a42e-5627-4a16-b6d6-0055b6604c11">

Reply:
Got backend to add a fix for this, will be deployed today